### PR TITLE
Add supported languages UI

### PR DESCRIPTION
**Pull request by bsudekum**
todo:
- [x] add note about avspeechsynth supporting most languages
- [ ] add instructions on how to add UI support
- [x] add note on how we dont have control over polly

/cc @mapbox/navigation-ios @geografa

### CODE REVIEW

File: languages.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Supported languages
+
+
+| Language | UI Elements  | PollyVoiceController  |
+|----------|--------------|-----------------------|
+| `ca` | ✅ | ❌ |
+| `de` | ✅ | ✅ |
+| `en` | ✅ | ✅ |
+| `es` | ❌ | ✅ |
```
Comment (1ec5) [inline languages.md]: https://github.com/mapbox/mapbox-navigation-ios/tree/master/MapboxNavigation/Resources/es.lproj

**Comment by bsudekum** [comment]
@1ec5 or @frederoni, do you think you could add some instructions on how to add/update a language with transifex?

File: languages.md [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
-| `pl` | ❌ | ✅ |
-| `ro` | ❌ | ✅ |
-| `ru` | ✅ | ✅ |
-| `sv` | ✅ | ✅ |
-| `tr` | ❌ | ✅ |
-| `vi` | ✅ | ❌ |
-| `zh` | ✅ | ❌ |
+The `PollyVoiceController` is powered by Amazon Web Service's Polly product. If Polly supports a language, we will add support for it in [`PollyVoiceController`](https://github.com/mapbox/mapbox-navigation-ios/blob/1d74296aa4c6adc779193fad07f0c97de2f79e90/MapboxNavigation/PollyVoiceController.swift#L99). Reference [AWS Polly supported languages](https://docs.aws.amazon.com/polly/latest/dg/SupportedLanguage.html).
+
+When a language is not supported by Polly, the SDK falls back to iOS's built in speech synthesizer, [AVSpeechSynthesizer](https://developer.apple.com/documentation/avfoundation/avspeechsynthesizer) which should support most languages.
```
Comment (1ec5) [inline languages.md]: This SDK automatically matches the application’s language when possible. The developer has no control over the navigation UI or speech synthesis language specifically. To enable one of the languages above, the developer needs to localize their application into that language. Even a stub Localizable.strings file will work.

Comment (1ec5) [inline languages.md]: Also link to [this documentation](https://www.mapbox.com/api-documentation/#instructions-languages) on instruction languages.

Comment (1ec5) [inline languages.md]: Note that street and destination names – as displayed on the map and UI and heard in spoken instructions – are currently limited to the `name` tag in OpenStreetMap, which is generally the local or national language. (In some regions, the name may include multiple languages or scripts. Throughout India, street and destination names are in English, regardless of the local language.)

**Comment by 1ec5** [comment]
> do you think you could add some instructions on how to add/update a language with transifex?

I started drafting localization instructions awhile back but neglected to PR them – thanks for the reminder. Does #929 include what you had in mind?

**Comment by bsudekum** [comment]
Yeah this looks great.

**Pull request merged**